package.json

```diff
@@ -12,9 +12,12 @@
     "babel-preset-es2015": "^6.24.0",
     "babel-register": "^6.24.0",
     "coffee-script": "~1.8.0",
+    "cookiejar": "^2.1.1",
+    "csprng": "^0.1.2",
```

Comment:
`csprng` and `cookiejar`? why do we need them as new deps?
and if we do - what connection does it have with this PR

Reply:
@zelima these libraries I included since when you run hubot locally, it will ask for them, and every time I installed manually.

Reply:
@Mikanebu I do not get you, it asks to be installed locally but not on Heroku?

Even so, I would at least make it a separate commit as it has no relation with this PR.  But ok, let's not bother for now with that.